ldpd/adjacency.c

```diff
@@ -386,7 +386,9 @@ adj_to_ctl(struct adj *adj)
 	}
 	actl.holdtime = adj->holdtime;
 	actl.holdtime_remaining =
-	    thread_timer_remain_second(adj->inactivity_timer);
+		thread_is_scheduled(adj->inactivity_timer)
```

Comment:
Why wouldn't we just return 0 if the thread is not scheduled in thread.c?  Won't we have other crashes conceivably because of this pattern?

Reply:
Yeah, I thought about this as well. Let's fix it here or with another PR (+ removing checks against NULL)?

Reply:
another pr is fine